tests/mock_gui/app.py

```diff
@@ -0,0 +1,13 @@
+import pyblish.cli
```

Comment:
Didn't know exact where you wanted this to happen, so I have made a context within the test module for creating the mock gui.
Do you think it should be per test instead of reusing it across multiple tests?